zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupConfig.java

```diff
@@ -22,7 +22,7 @@
  * Backup-related configurations
  */
 public class BackupConfig {
-  public static final String NFS_BACKUP_STORAGE = "NFSBackupStorage";
+  public static final String NFS_BACKUP_STORAGE = "NfsBackupStorage";
```

Comment:
I don't think this is the appropriate place for this constant. BackupConfig is a general config POJO for all types of backup implementations. 

A more appropriate place for this constant would be NfsBackupStorage implementation.

Reply:
Good catch. Actually it turned out that I don't need this constant since we are using the Java class names.